packages/@okta/vuepress-site/docs/guides/terraform-oig-resources/main/entitlement-bundles/create.md

```diff
@@ -0,0 +1,9 @@
+
+
```

Comment:
These "create" examples don't seem correct as they don't have the minimal required parameters.